Sources/Danger/GitHubDSL.swift

```diff
@@ -17,489 +13,449 @@ public struct GitHub: Decodable {
-        case url
-        case author
-        case committer
-    }
+        /// Generic UUID.
+        public let id: Int
 
-    /// The SHA for the commit.
-    public let sha: String
+        /// The name of the repo, e.g. "danger-swift".
+        public let name: String
 
-    /// The raw commit metadata.
-    public let commit: GitCommit
+        /// The full name of the owner + repo, e.g. "Danger/danger-swift"
+        public let fullName: String
 
-    /// The URL for the commit on GitHub.
-    public let url: String
+        /// The owner of the repo.
+        public let owner: User
 
-    /// The GitHub user who wrote the code.
-    public let author: GitHubUser?
+        /// A boolean stating whether the repo is publicly accessible.
+        public let isPrivate: Bool
 
-    /// The GitHub user who shipped the code.
-    public let committer: GitHubUser?
+        /// A textual description of the repo.
+        public let description: String?
 
-    public init(from decoder: Decoder) throws {
-        let container = try decoder.container(keyedBy: CodingKeys.self)
+        /// A boolean stating whether the repo is a fork.
+        public let isFork: Bool
 
-        sha = try container.decode(String.self, forKey: .sha)
-        commit = try container.decode(GitCommit.self, forKey: .commit)
-        url = try container.decode(String.self, forKey: .url)
-        author = (try? container.decodeIfPresent(GitHubUser.self, forKey: .author)) ?? nil
-        committer = (try? container.decodeIfPresent(GitHubUser.self, forKey: .committer)) ?? nil
+        /// The root web URL for the repo, e.g. https://github.com/artsy/emission
+        public let htmlURL: String
     }
 }
 
-// MARK: - GitHubIssue
-
-public struct GitHubIssue: Decodable, Equatable {
-    // MARK: - CodingKeys
-
-    enum CodingKeys: String, CodingKey {
-        case id
-        case number
-        case title
-        case user
-        case state
-        case assignee
-        case assignees
-        case milestone
-        case body
-        case labels
-        case commentCount = "comments"
-        case isLocked = "locked"
-        case createdAt = "created_at"
-        case updatedAt = "updated_at"
-        case closedAt = "closed_at"
+extension GitHub {
+    public struct Review: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
+            case user
+            case id
+            case body
+            case state
+            case commitId = "commit_id"
+        }
+
+        public enum State: String, Decodable {
+            case approved = "APPROVED"
+            case requestedChanges = "CHANGES_REQUESTED"
+            case comment = "COMMENTED"
+            case pending = "PENDING"
+            case dismissed = "DISMISSED"
+        }
+
+        /// The user who has completed the review or has been requested to review.
+        public let user: User
+
+        /// The id for the review (if a review was made).
+        public let id: Int?
```

Comment:
<!--
  0 failure: 
  1 warning:  Variable name sho...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 264;
-->

- :warning: Variable name should be between 3 and 40 characters long: 'id' (`identifier_name`)